src/Models/ContactFollowup.php

```diff
@@ -0,0 +1,20 @@
+<?php
```

Comment:
@drewroberts I built this out as a model, is this the approach you want me to take or keep it at simple as just a migration called `create_contact_followups`. I didn't build out everything for this model if so until I had your word on this.

Reply:
Yes, that's perfect. Go ahead and create a factory, policy & nova resource.

I'm still not sold on the name "followups" - maybe we should use contact_responses instead. That would make more sense if we add communication between the contact and our staff and pull in their response back to a question.

Reply:
Renamed to `ContactResponse` in https://github.com/tipoff/forms/pull/43/commits/b1fb50cb8ecac07c5b8ad6dec3f308f8d72d8d92